examples/preprocessing/eeg_bridging.py

```diff
@@ -0,0 +1,255 @@
+"""
+.. _ex-eeg-bridging:
+
+===============================================
+Identify EEG Electrodes Bridged by too much Gel
+===============================================
+
+Research-grade EEG often uses a gel based system, and when too much gel is
+applied the gel conducting signal from the scalp to the electrode for one
+electrode connects with the gel conducting signal from another electrode
+"bridging" the two signals. This is undesirable because the signals from the
+two (or more) electrodes are not as independent as they would otherwise be;
+they are more similar to each other than they would otherwise be causing
+spatial smearing. An algorithm has been developed to detect electrode
+bridging :footcite:`TenkeKayser2001`, which has been implemented in EEGLAB
+:footcite:`DelormeMakeig2004`. Unfortunately, there is not a lot to be
+done about electrode brigding once the data has been collected as far as
+preprocessing. Therefore, the recommendation is to check for electrode
+bridging early in data collection and address the problem. Or, if the data
+has already been collected, quantify the extent of the bridging so as not
+to introduce bias into the data from this effect and exclude subjects with
+bridging that might effect the outcome of a study. Preventing electrode
+bridging is ideal but awareness of the problem at least will mitigate its
+potential as a confound to a study. This tutorial follows
+https://psychophysiology.cpmc.columbia.edu/software/eBridge/tutorial.html.
+"""
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+# sphinx_gallery_thumbnail_number = 2
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+
+print(__doc__)
+
+# %%
+# Compute Electrical Distance Metric
+# ----------------------------------
+# First, let's compute electrical distance metrics for a group of example
+# subjects from the EEGBCI dataset in order to estimate electrode bridging.
+# The electrical distance is just the variance of signals subtracted
+# pairwise. Channels with activity that mirror another channel nearly
+# exactly will have very low electrical distance. By inspecting the
+# distribution of electrical distances, we can look for pairwise distances
+# that are consistently near zero which are indicative of bridging.
+#
+# .. note:: It is likely to be sufficient to run this algorithm on a
+#           small portion (~3 minutes is probably plenty) of the data but
+#           that gel might settle over the course of a study causing more
+#           bridging so using the last segment of the data will
+#           give the most conservative estimate.
+
+ed_data = dict()  # electrical distance/bridging data
+raw_data = dict()  # store infos for electrode positions
+for sub in range(1, 11):
+    print(f'Computing electrode bridges for subject {sub}')
+    raw = mne.io.read_raw(mne.datasets.eegbci.load_data(
+        subject=sub, runs=(1,))[0], preload=True, verbose=False)
```

Comment:
might be a little clearer to separate these nested calls into two separate lines